pkg/skaffold/initializer/init_test.go

```diff
@@ -28,23 +28,31 @@ import (
 func TestPrintAnalyzeJSON(t *testing.T) {
 	tests := []struct {
 		description string
-		dockerfiles []InitBuilder
+		pairs       []builderImagePair
+		builders    []InitBuilder
 		images      []string
 		skipBuild   bool
 		shouldErr   bool
 		expected    string
 	}{
 		{
-			description: "dockerfile and image",
-			dockerfiles: []InitBuilder{docker.Docker("Dockerfile1"), docker.Docker("Dockerfile2")},
+			description: "builders and images with pairs",
+			pairs:       []builderImagePair{{docker.Docker{Dockerfile: "Dockerfile1"}, "image1"}},
+			builders:    []InitBuilder{docker.Docker{Dockerfile: "Dockerfile2"}},
+			images:      []string{"image2"},
+			expected:    "{\"builders\":[{\"name\":\"Docker\",\"payload\":{\"path\":\"Dockerfile1\"}},{\"name\":\"Docker\",\"payload\":{\"path\":\"Dockerfile2\"}}],\"images\":[{\"name\":\"image1\",\"requiresPrompt\":false},{\"name\":\"image2\",\"requiresPrompt\":true}]}",
```

Comment:
These strings would be easier to read if you use backticks. This way you don't have to escape quotes.